.goreleaser.yml

```diff
@@ -607,6 +607,211 @@ builds:
 ## End of Linux AMD64
 
 
+## Linux AMD64-v2:
```

Comment:
Is it possible to parametrize the logic on v1, v2 so that we avoid the copy-paste?

Reply:
Good idea :+1:
I am going to play with templates.

Reply:
Goreleaser configuration block for amd64 and amd64/v2 is now the same template for both -- which makes it simply to manage in the future.
Please, review changes, @yperbasis .